test/common/secret/secret_manager_impl_test.cc

```diff
@@ -1128,6 +1128,36 @@ name: "abc.com"
       EnvoyException, "Failed to load private key provider: test");
 }
 
+// Test that we don't allow specification of both typed and untyped matchers for
+// sans.
+TEST_F(SecretManagerImplTest, DeprecatedSanMatcher) {
```

Comment:
This test should probably be in `test/extensions/transport_sockets/tls`

Reply:
Moved test.